cap/worker_options.go

```diff
@@ -0,0 +1,120 @@
+package cap
+
+import (
+	"github.com/capatazlib/go-capataz/internal/c"
+)
+
+// Restart specifies when a goroutine gets restarted
+type Restart = c.Restart
+
+// Permanent specifies that a goroutine should be restarted whether or not there
+// are errors.
+//
+// You can specify this option using the WithRestart function
+var Permanent = c.Permanent
+
+// Transient specifies that a goroutine should be restarted if and only if the
+// goroutine failed with an error. If the goroutine finishes without errors it
+// is not restarted again.
+//
+// You can specify this option using the WithRestart function
+var Transient = c.Transient
+
+// Temporary specifies that the goroutine should not be restarted under any
+// circumstances
+//
+// You can specify this option using the WithRestart function
+var Temporary = c.Temporary
+
+// Shutdown is an enum type that indicates how the parent supervisor will handle
+// the stoppping of the worker goroutine
+//
+type Shutdown = c.Shutdown
+
+// Indefinitely is a Shutdown value that specifies the parent supervisor must
+// wait indefinitely for the worker goroutine to stop executing
+//
+// You can specify this option using the WithShutdown function
+var Indefinitely = c.Indefinitely
+
+// Timeout is a Shutdown function that returns a value that indicates the time
+// that the supervisor will wait before "force-killing" a worker goroutine. This
+// function receives a time.Duration value
+//
+// You can specify this option using the WithShutdown function
+//
+// * Warning
```

Comment:
Are you expecting special formatting or just the `*`?

Reply:
Nope